### DISCUSSION TITLE
Support clone only selected files

### DISCUSSION THREAD

**Discussion by RonaldGuilhermePDS** [community/community#18939] [Actions] [answered]
It would be very useful if we could clone only selected files from the github repository. what do you think about it?

**Comment by thuongtruong1009** [comment, marked as answer]
Hi @RonaldGuilhermePDS 
* Firstly, you go to a selected file that you want to clone and click to open it.
* Then, you click to raw button on the right and you will get a URL like https://raw.githubusercontent.com/current_username/current_pository_name/current_branch/current_file)
* Finally, you only press Ctrl+S. And the file will be downloaded to local successfully.

##### Example: 
 * I want to download a file name ***tsconfig.json*** at repo https://github.com/thuongtruong1009/e-shopee/blob/main/tsconfig.json
    ***Step1***: I click the raw button on the right,  like that 
     ![image](https://user-images.githubusercontent.com/71834167/174526741-74aea6e2-9796-4b2c-a590-4c105b0f865f.png)
    ***Step2***: You press ***Ctrl+S*** and download to local
###### Hope it useful for you!
###### Have a nice day!
